tests/rscript_parse_time_get-ts.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
```

Comment:
for the non-pythonic (and as an aid in any case), could you add a little doc of what the script actually does - same for the test. I think I get the gist - do an eval that the test driver itself is OK, then inject the data, but it would be much easier to spell it out IMHO. I think this is important as the method is quite a bit different from the regular way tests operate.

Reply:
Done. Let me know if that seems clear enough.